### ISSUE TITLE
disable-registration but allow invitational register

### ISSUE THREAD

**Issue by Mattchewone** [penpot/penpot#5178]
### Steps To Reproduce

1. use these flags in the frontend & backend
```disable-registration enable-login-with-password```

2. With an existing user, send an invite to a user
3. Get the link from the logs and use the link
4. Unable to register a new user with the link

### Expected behavior

As per the docs in the flags section, I would expect an invitation to allow a user registration

```
disable-registration: disables registration (still enabled for invitations only).
```

### Actual behavior

There is no ability to create an account or register when using an invitation link

**Comment by scdigitalmedia** [comment]
I'm having this issue as well. From what I understand I can disable-registration but still allow invite emails to work. however the invite link is going to the login page instead of the register page.

**Comment by violoncelloCH** [comment]
exactly; this worked in an earlier version, but not any more as it seems

**Comment by K-MeLeOn** [comment]
Got this behavior too 👍

**Comment by zackrspv** [comment]
Same, sadly, looks like is not working in this updated docker version.  Still trying different options tho.  Note, one could enable registration but restrict to domains :)

**Comment by johnrah** [comment]
Without the ability to invite clients, the tool is unusable.  It's only good for internal use. Hopefully they can fix this soon.

**Comment by deseven** [comment]
As a workaround you can manually create users using CLI:  
https://help.penpot.app/technical-guide/getting-started/#using-the-cli-for-administrative-tasks

**Comment by EncryptedData** [comment]
Is there any update on this? I am self-hosting this for a few friends, and I want them to be able to invite users without me creating users.

**Comment by iainsmacleod** [comment]
Love to see any update on this, happy to help and provide any logs.
